app/src/main/java/org/schabi/newpipe/util/DeviceUtils.java

```diff
@@ -35,6 +36,18 @@
     private DeviceUtils() {
     }
 
+    public static boolean isFireTv() {
+        if (isFireTV != null) {
+            return isFireTV;
+        }
+
+        final boolean isFireTv =
```

Comment:
why do you create a new variable?

Reply:
I was just following the pattern of `isTv()` below. If you think it's not needed I can remove it.